bentoml/_internal/utils/dataframe.py

```diff
@@ -30,37 +30,37 @@ def check_dataframe_column_contains(
 def guess_orient(
     table: t.Union[t.List[t.Mapping[str, t.Any]], t.Dict[str, t.Any]],
     strict: bool = False,
-) -> t.Optional[t.Union[str, t.Set[str]]]:
+) -> t.Optional[t.Set[str]]:
```

Comment:
are the changes here used?

Reply:
You can't assign type Union to a variable, so in some consequent function using `guess_orient` variable should just have one type instead of two. Mypy can't seem to detect it automatically